src/app/csc/csc.component.ts

```diff
@@ -92,32 +78,24 @@ export class CscComponent implements OnInit {
     private location: Location,
     private aroute: ActivatedRoute,
     private urlService: UrlService,
-  ) {}
-
-  showAllProjects() {
-    this.filteredCscProjectsList = [];
-    for (var project in this.cscProjectsList) {
-      this.filteredCscProjectsList.push(this.cscProjectsList[project]);
-    }
+    private cdr: ChangeDetectorRef,
+  ) {
+    this.dataSource = new MatTableDataSource<any>();
   }
 
-  sortProjectsByKey(array, key) {
-    return array.sort(function (a, b) {
```

Comment:
yessssss bye

Reply:
I thought you might be happy about that 😄